pkg/controller/node/nodecontroller.go

```diff
@@ -187,6 +187,7 @@ func NewNodeController(
 			AddFunc:    nc.maybeDeleteTerminatingPod,
 			UpdateFunc: func(_, obj interface{}) { nc.maybeDeleteTerminatingPod(obj) },
 		},
+		cache.Indexers{},
```

Comment:
Can you please add comment why you are not creating any indexes here?

[Again, I think it's fine here, but please be explicit about it].